apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx

```diff
@@ -35,7 +35,7 @@ async function getAIResponse(messages) {
     temperature: 0.7,
   };
 
-  const response = await chatService.chat.completions.create(params);
+  const response = await chatService.chat.completions.create(params as any);
```

Comment:
I think we should avoid 'any' and assertions as untipatterns in our demos
We can define params using required type, add examples with some artificial type - you can change it for the required one 

const params: ChatServiceCreateParams = {
    messages,
    max_tokens: 1000,
    temperature: 0.7,
  };

Reply:
i did it to avoid required `model` passing
<img width="889" alt="image" src="https://github.com/user-attachments/assets/d8bad573-5e1f-4611-9bb7-c14e45ed51ad">
but ok i guess it's ok to use the one we use in service config